CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,9 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Proxy] Fix non-relevant throw of the error "operation timed out" on (re)deploying a proxy [#4927](https://github.com/vatesfr/xen-orchestra/issues/4927) (PR [#4928](https://github.com/vatesfr/xen-orchestra/pull/4928))
```

Comment:
What do you mean by *non-relevant*?

Reply:
Non-pertinent error.

Reply:
What information does this provide to the reader?

Reply:
We have fixed some cases when the error "operation timed out" isn't pertinent.

Reply:
This is a vague explanation, would it be possible to find a better one?

Reply:
The user will only see the error `operation timed out` on deploying a proxy.

Reply:
Let's simplify the message then:

```suggestion
- [Proxy] Fix `operation timed out` error on (re)deploy [#4927](https://github.com/vatesfr/xen-orchestra/issues/4927) (PR [#4928](https://github.com/vatesfr/xen-orchestra/pull/4928))
```

Reply:
IMHO, the most explicit explanation is this https://github.com/vatesfr/xen-orchestra/pull/4928#discussion_r410074736

Reply:
I don't putted this message because this error can happens.